web/src/message_events.js

```diff
@@ -380,7 +380,7 @@ export function update_messages(events) {
         const stream_changed = new_stream_id !== undefined;
         const stream_archived = old_stream === undefined;
 
-        if (!topic_edited && !stream_changed) {
+        if (anchor_message !== undefined && !topic_edited && !stream_changed) {
             // If the topic or stream of the message was changed,
             // it will be rerendered if present in any rendered list.
             messages_to_rerender.push(anchor_message);
```

Comment:
Hmm, but do we want content edits where the anchor message isn't cached locally to go into that else clause for moving topics? I don't think so.

Reply:
oh yeah! Thanks!